tests/migration_test.go

```diff
@@ -1022,6 +1022,90 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}
 				Expect(timerFrequency).ToNot(BeEmpty())
 			})
+			findTwoNodesWithCloseOrFarTSCFreqencies := func(nodes *k8sv1.NodeList, close bool) (string, string) {
+				for _, node1 := range nodes.Items {
+					freq1, exists := node1.Labels[topology.TSCFrequencyLabel]
+					if !exists {
+						continue
+					}
+					f1, err := strconv.ParseInt(freq1, 10, 64)
+					ExpectWithOffset(1, err).ToNot(HaveOccurred())
+					for _, node2 := range nodes.Items {
+						freq2, exists := node2.Labels[topology.TSCFrequencyLabel]
+						if !exists {
+							continue
+						}
+						if freq1 == freq2 {
+							continue
+						}
+						f2, err := strconv.ParseInt(freq2, 10, 64)
+						ExpectWithOffset(1, err).ToNot(HaveOccurred())
+						diff := f1 - f2
+						if diff < 0 {
+							diff = -diff
+						}
+						if close && diff <= 533000 {
+							return node1.Name, node2.Name
+						}
+						if !close && diff > 533000 {
+							return node1.Name, node2.Name
+						}
+					}
+				}
+				return "", ""
+			}
+			cordonAllBut := func(nodes *k8sv1.NodeList, node1, node2 string) {
+				for _, node := range nodes.Items {
+					if node.Name != node1 && node.Name != node2 {
+						libnode.Taint(node.Name, libnode.GetNodeDrainKey(), k8sv1.TaintEffectNoSchedule)
+					}
+				}
+			}
+			It("should migrate to a node with a close TSC frequency", Serial, decorators.TscFrequencies, func() {
```

Comment:
Can you put both of these tests into a `DescribeTable`? They are very similar except for the call to `findTwoNodesWithCloseOrFarTSCFreqencies` and the skip message.